### PR TITLE
Improve docs for Building from source

### PR DESCRIPTION
**Pull request by satya164**
cc @ide

### CODE REVIEW

File: docs/AndroidBuildingFromSource.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ ndk.dir=/Users/your_unix_name/android-ndk/android-ndk-r10e
 
 ## Building the source
 
-1. Install `react-native` from your fork. For example, to install the master branch from the offcial repo, run the following:
+#### 1. Installing the fork
+
+Frst, you need to install `react-native` from your fork. For example, to install the master branch from the offcial repo, run the following:
```
Comment (ide) [inline docs/AndroidBuildingFromSource.md]: fix spelling

**Comment by satya164** [comment]
@ide done

**Comment by qbig** [comment]
@satya164 @ide @mkonicek Could give some pointers on how to enable debugging with ndk for the cpp implementations ? I attempted to walk through, but all the break points in cpp code were ignored.

**Comment by satya164** [comment]
Sorry @qbig . I've zero clue on this.

**Pull request merged**